src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/AppViewModel.kt

```diff
@@ -3,25 +3,37 @@ package org.vaccineimpact.orderlyweb.viewmodels
 import org.pac4j.core.profile.CommonProfile
 import org.vaccineimpact.orderlyweb.ActionContext
 import org.vaccineimpact.orderlyweb.db.AppConfig
+import org.vaccineimpact.orderlyweb.db.Config
+import org.vaccineimpact.orderlyweb.db.MissingConfigurationKey
 import org.vaccineimpact.orderlyweb.security.authentication.AuthenticationConfig
 
 data class Breadcrumb(val name: String, val url: String?)
 
 data class DefaultViewModel(override val loggedIn: Boolean,
                        override val user: String?,
-                       override val breadcrumbs: List<Breadcrumb>) : AppViewModel
+                       override val breadcrumbs: List<Breadcrumb>,
+                       private val appConfig: Config = AppConfig()) : AppViewModel
 {
     constructor(userProfile: CommonProfile?, breadcrumbs: List<Breadcrumb>) :
             this(userProfile != null, userProfile?.id, breadcrumbs)
 
     constructor(context: ActionContext, vararg breadcrumbs: Breadcrumb) :
             this(context.userProfile, breadcrumbs.toList())
 
-    override val appName = AppConfig()["app.name"]
-    override val appEmail = AppConfig()["app.email"]
+    override val appName = appConfig["app.name"]
+    override val appEmail = appConfig["app.email"]
     override val authProvider = AuthenticationConfig().getConfiguredProvider().toString()
-    override val logo = AppConfig()["app.logo"]
-
+    override val logo = appConfig["app.logo"]
+    override val montaguApiUrl =
+            try
+            {
+                appConfig["montagu.client_api_url"]
```

Comment:
Annoyingly, now that we're sometimes accessing the montagu api from the server (to get user details) and sometimes from the client (to reset cookies on logout), we need to use different urls only if testing locally through the proxy ('localhost' doesn't work from within the docker container. montagu.client_api_url is only set in the dev config properties in the proxy). Also, see note on that branch on circumventing browser invalid cert error.

Reply:
Urgh, this is annoying. I wonder if a better solution would just be to use the host network when testing the proxy, so that localhost refers to the right thing? This feels like an added complication and a more confusing configuration interface that it would be better to avoid. If we map everything to the host network when testing the proxy it's actually a closer analogue of the real situation right, just with `localhost` instead of `support.montagu..` or `montagu.vaccineimapct...`?

Reply:
That would mean just changing the network to 'host' in the proxy's docker-compose.yml? (Compose is only used for testing I think?) I'll try it and see if it works.